region/norfair/west/Crocomire Speedway.json

```diff
@@ -253,14 +253,14 @@
       "notable": true,
       "entranceCondition": {
         "comeInShinecharged": {
-          "framesRequired": 60
+          "framesRequired": 35
         }
       },
       "requires": [
         "h_canNavigateHeatRooms",
         "canShinechargeMovement",
-        {"shinespark": {"frames": 84, "excessFrames": 10}},
-        {"heatFrames": 760}
+        {"shinespark": {"frames": 86, "excessFrames": 10}},
+        {"heatFrames": 700}
```

Comment:
After the spark you right right then back while blue in order to break the rest of the speed blocks. Is using spring ball, to get over the speed blocks, better if you have it?

Reply:
I gave it a try but it looks like no, even in best case scenario it takes almost a second longer to try to get through with Spring Ball.